### PR TITLE
Adding map feature in app

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #80 
Currently map shows location of FossAsia office singapore by adding in moreViewController
image

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@chashmeetsingh @imjog please review pr!!

File: FOSSAsia/MapViewController.swift [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import MapKit
+import CoreLocation
+
+class MapViewController: UIViewController {
+
+    @IBOutlet weak var mapView: MKMapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        let locationCoordinates = CLLocationCoordinate2DMake(Constants.Location.lattitude, Constants.Location.longitude)
```
Comment (jogendra) [inline FOSSAsia/MapViewController.swift:19]: Why is the location coordinate hard-coded? I think we are getting location coordinate from server side.

Comment (SIMRAN1) [reply FOSSAsia/MapViewController.swift:19]: @imjog I have added map to show location for fossAsia office only. I am studying the backend api part of it!!

**Comment by SIMRAN1** [comment]
@chashmeetsingh please review the feature!

**Review by chashmeetsingh — APPROVED** [review]

**Review by jogendra — APPROVED** [review]

**Pull request merged**